docs/csharp/debugging.md

```diff
@@ -200,17 +200,17 @@ With the C# Dev Kit extension installed, the debugger allows you to apply C# cod
 
 ![Hot Reload displayed in the debugging toolbar](images/debugging/hotreload-toolbar.png)
 
-In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [User Settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
+In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [user settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
 
-### Supported .NET apps and scenarios
+### Supported projects and scenarios
```

Comment:
When you change an H2 or H3 header, you should check for direct links. In this case, search for "supported-net-apps-and-scenarios" and update to "supported-projects-and-scenarios"
I see at least one link in the FAQ to this header